CHANGELOG.md

```diff
@@ -7,6 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add support for `monitoring_config` block
+- Add support for `managed_prometheus` block inside `monitoring_config`
+
+### Removed
+
+- BREAKING CHANGE: Remove variable `monitoring_enable_components`
+- BREAKING CHANGE: Remove `var.module_enabled` output
+
+  ###### old format
+
+  ```
+  monitoring_enable_components = ["SYSTEM_COMPONENTS", "WORKLOADS"]
+  ```
+
+  ###### new format
+  ```
```

Comment:
```suggestion
```

Reply:
this breaks the formatting:
<img width="1419" alt="Screenshot 2022-10-25 at 12 20 58" src="https://user-images.githubusercontent.com/10109428/197751853-de7e3b32-b8b4-475c-981c-576a20355693.png">
<img width="1415" alt="Screenshot 2022-10-25 at 12 21 40" src="https://user-images.githubusercontent.com/10109428/197751862-cedafc94-b106-496b-82c7-8f88fcba3bb0.png">

Reply:
wdyt? double ``` does not make any sense... seems you have different characters...